providers/azure/aci.go

```diff
@@ -51,6 +51,28 @@ type AuthConfig struct {
 	RegistryToken string `json:"registrytoken,omitempty"`
 }
 
+const westEuropeRegion = "westeurope"
```

Comment:
can you make it an array legalAciRegions? we can add region easily later. right now, we need to update 3 places to add a new region.

otherwise LGTM

Reply:
Latest change should only require making any updates to the 'validAciRegions' array.